installer/scripts/omsadmin.sh

```diff
@@ -95,6 +95,7 @@ PROC_LIMIT_CONF=/etc/security/limits.conf
 # SCOM variables
 SCX_SSL_CONFIG=/opt/microsoft/scx/bin/tools/scxsslconfig
 OMI_CONF_FILE=/etc/opt/omi/conf/omiserver.conf
+OMI_CONF_EDITOR=/opt/omi/bin/omiconfigeditor
```

Comment:
That is true. @vimish added this to OMI v1.3. So OMS now has a minimum dependency of OMS v1.3 since it depends on this capability. @vimish, I'm going to mark my review "Waiting" for this change.